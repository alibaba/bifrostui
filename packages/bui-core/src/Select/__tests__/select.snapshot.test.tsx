import { snapshotTest } from 'testing';

describe('Select snapshot', () => {
  snapshotTest('Select');
});
