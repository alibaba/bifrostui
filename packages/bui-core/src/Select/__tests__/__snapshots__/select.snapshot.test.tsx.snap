// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select snapshot Select demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "stretch",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "background": "#eee",
      "padding": "50px",
    }
  }
>
  <div
    className="bui-select"
    onClick={[Function]}
  >
    <div
      className="bui-select-selector-container"
    >
      <div
        className="bui-select-selector"
      />
      <input
        className="bui-select-input"
        readOnly={true}
        value=""
      />
      <svg
        className="bui-svg-icon bui-select-selector-icon"
        color="#9c9ca5"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M50 58.1c-.1.2-.3.3-.5.5-1.1.8-2.6.6-3.4-.5L32.5 40.8c-.3-.4-.5-.9-.5-1.4 0-1.3 1.1-2.4 2.5-2.4h27.1c.5 0 1.1.2 1.5.5 1.1.8 1.3 2.3.5 3.3L50 58.1z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-select-option-container"
      style={
        Object {
          "WebkitTransition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "opacity": "0",
          "transition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "visibility": "hidden",
        }
      }
    >
      <div
        className="bui-select-option-main"
        style={
          Object {
            "WebkitTransform": "translateY(-100%)",
            "WebkitTransition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "transform": "translateY(-100%)",
            "transition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "visibility": "hidden",
          }
        }
      >
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 1
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 2
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 3
        </div>
      </div>
    </div>
  </div>
  <div
    className="bui-backdrop bui-backdrop-invisible bui-select-backdrop"
    onClick={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTransition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "opacity": "0",
        "transition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "visibility": "hidden",
      }
    }
  />
</div>
`;

exports[`Select snapshot Select demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "stretch",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "background": "#eee",
      "padding": "50px",
    }
  }
>
  <div
    className="bui-select"
    onClick={[Function]}
  >
    <div
      className="bui-select-selector-container"
    >
      <div
        className="bui-select-selector"
      >
        下拉选择
      </div>
      <input
        className="bui-select-input"
        readOnly={true}
        value=""
      />
      <svg
        className="bui-svg-icon bui-select-selector-icon"
        color="#9c9ca5"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M50 58.1c-.1.2-.3.3-.5.5-1.1.8-2.6.6-3.4-.5L32.5 40.8c-.3-.4-.5-.9-.5-1.4 0-1.3 1.1-2.4 2.5-2.4h27.1c.5 0 1.1.2 1.5.5 1.1.8 1.3 2.3.5 3.3L50 58.1z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-select-option-container"
      style={
        Object {
          "WebkitTransition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "opacity": "0",
          "transition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "visibility": "hidden",
        }
      }
    >
      <div
        className="bui-select-option-main"
        style={
          Object {
            "WebkitTransform": "translateY(-100%)",
            "WebkitTransition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "transform": "translateY(-100%)",
            "transition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "visibility": "hidden",
          }
        }
      >
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 1
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 2
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 3
        </div>
      </div>
    </div>
  </div>
  <div
    className="bui-backdrop bui-backdrop-invisible bui-select-backdrop"
    onClick={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTransition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "opacity": "0",
        "transition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "visibility": "hidden",
      }
    }
  />
</div>
`;

exports[`Select snapshot Select demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "stretch",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "background": "#eee",
      "padding": "50px",
    }
  }
>
  <div
    className="bui-select"
    onClick={[Function]}
  >
    <div
      className="bui-select-selector-container"
    >
      <div
        className="bui-select-selector"
      />
      <input
        className="bui-select-input"
        readOnly={true}
        value={2}
      />
      <svg
        className="bui-svg-icon bui-select-selector-icon"
        color="#9c9ca5"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M50 58.1c-.1.2-.3.3-.5.5-1.1.8-2.6.6-3.4-.5L32.5 40.8c-.3-.4-.5-.9-.5-1.4 0-1.3 1.1-2.4 2.5-2.4h27.1c.5 0 1.1.2 1.5.5 1.1.8 1.3 2.3.5 3.3L50 58.1z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-select-option-container"
      style={
        Object {
          "WebkitTransition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "opacity": "0",
          "transition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "visibility": "hidden",
        }
      }
    >
      <div
        className="bui-select-option-main"
        style={
          Object {
            "WebkitTransform": "translateY(-100%)",
            "WebkitTransition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "transform": "translateY(-100%)",
            "transition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "visibility": "hidden",
          }
        }
      >
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 1
        </div>
        <div
          className="bui-select-option bui-select-option-active"
          onClick={[Function]}
        >
          option 2
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 3
        </div>
      </div>
    </div>
  </div>
  <div
    className="bui-backdrop bui-backdrop-invisible bui-select-backdrop"
    onClick={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTransition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "opacity": "0",
        "transition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "visibility": "hidden",
      }
    }
  />
</div>
`;

exports[`Select snapshot Select demo snapshot 0 4`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "stretch",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "20px",
      "background": "#eee",
      "padding": "50px",
    }
  }
>
  <div
    className="bui-select"
    onClick={[Function]}
  >
    <div
      className="bui-select-selector-container"
    >
      <div
        className="bui-select-selector"
      >
        选择器A
      </div>
      <input
        className="bui-select-input"
        readOnly={true}
        value=""
      />
      <svg
        className="bui-svg-icon bui-select-selector-icon"
        color="#9c9ca5"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M50 58.1c-.1.2-.3.3-.5.5-1.1.8-2.6.6-3.4-.5L32.5 40.8c-.3-.4-.5-.9-.5-1.4 0-1.3 1.1-2.4 2.5-2.4h27.1c.5 0 1.1.2 1.5.5 1.1.8 1.3 2.3.5 3.3L50 58.1z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-select-option-container"
      style={
        Object {
          "WebkitTransition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "opacity": "0",
          "transition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "visibility": "hidden",
        }
      }
    >
      <div
        className="bui-select-option-main"
        style={
          Object {
            "WebkitTransform": "translateY(-100%)",
            "WebkitTransition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "transform": "translateY(-100%)",
            "transition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "visibility": "hidden",
          }
        }
      >
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 1
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 2
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 3
        </div>
      </div>
    </div>
  </div>
  <div
    className="bui-backdrop bui-backdrop-invisible bui-select-backdrop"
    onClick={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTransition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "opacity": "0",
        "transition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "visibility": "hidden",
      }
    }
  />
  <div
    className="bui-select"
    onClick={[Function]}
  >
    <div
      className="bui-select-selector-container"
    >
      <div
        className="bui-select-selector"
      >
        选择器B
      </div>
      <input
        className="bui-select-input"
        readOnly={true}
        value=""
      />
      <svg
        className="bui-svg-icon bui-select-selector-icon"
        color="#9c9ca5"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M50 58.1c-.1.2-.3.3-.5.5-1.1.8-2.6.6-3.4-.5L32.5 40.8c-.3-.4-.5-.9-.5-1.4 0-1.3 1.1-2.4 2.5-2.4h27.1c.5 0 1.1.2 1.5.5 1.1.8 1.3 2.3.5 3.3L50 58.1z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-select-option-container"
      style={
        Object {
          "WebkitTransition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "opacity": "0",
          "transition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "visibility": "hidden",
        }
      }
    >
      <div
        className="bui-select-option-main"
        style={
          Object {
            "WebkitTransform": "translateY(-100%)",
            "WebkitTransition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "transform": "translateY(-100%)",
            "transition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "visibility": "hidden",
          }
        }
      >
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 1
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 2
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 3
        </div>
      </div>
    </div>
  </div>
  <div
    className="bui-backdrop bui-backdrop-invisible bui-select-backdrop"
    onClick={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTransition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "opacity": "0",
        "transition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "visibility": "hidden",
      }
    }
  />
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
    onClick={[Function]}
    type="button"
  >
    提交
  </button>
</div>
`;

exports[`Select snapshot Select demo snapshot 0 5`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "stretch",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "20px",
      "background": "#eee",
      "padding": "50px",
    }
  }
>
  <div
    className="bui-select bui-select-disabled"
    onClick={[Function]}
  >
    <div
      className="bui-select-selector-container"
    >
      <div
        className="bui-select-selector"
      >
        禁用
      </div>
      <input
        className="bui-select-input"
        readOnly={true}
        value=""
      />
      <svg
        className="bui-svg-icon bui-select-selector-icon"
        color="#9c9ca5"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M50 58.1c-.1.2-.3.3-.5.5-1.1.8-2.6.6-3.4-.5L32.5 40.8c-.3-.4-.5-.9-.5-1.4 0-1.3 1.1-2.4 2.5-2.4h27.1c.5 0 1.1.2 1.5.5 1.1.8 1.3 2.3.5 3.3L50 58.1z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-select-option-container"
      style={
        Object {
          "WebkitTransition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "opacity": "0",
          "transition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "visibility": "hidden",
        }
      }
    >
      <div
        className="bui-select-option-main"
        style={
          Object {
            "WebkitTransform": "translateY(-100%)",
            "WebkitTransition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "transform": "translateY(-100%)",
            "transition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "visibility": "hidden",
          }
        }
      >
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 1
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 2
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 3
        </div>
      </div>
    </div>
  </div>
  <div
    className="bui-backdrop bui-backdrop-invisible bui-select-backdrop"
    onClick={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTransition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "opacity": "0",
        "transition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "visibility": "hidden",
      }
    }
  />
  <div
    className="bui-select"
    onClick={[Function]}
  >
    <div
      className="bui-select-selector-container"
    >
      <div
        className="bui-select-selector"
      />
      <input
        className="bui-select-input"
        readOnly={true}
        value={1}
      />
      <svg
        className="bui-svg-icon bui-select-selector-icon"
        color="#9c9ca5"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M50 58.1c-.1.2-.3.3-.5.5-1.1.8-2.6.6-3.4-.5L32.5 40.8c-.3-.4-.5-.9-.5-1.4 0-1.3 1.1-2.4 2.5-2.4h27.1c.5 0 1.1.2 1.5.5 1.1.8 1.3 2.3.5 3.3L50 58.1z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-select-option-container"
      style={
        Object {
          "WebkitTransition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "opacity": "0",
          "transition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "visibility": "hidden",
        }
      }
    >
      <div
        className="bui-select-option-main"
        style={
          Object {
            "WebkitTransform": "translateY(-100%)",
            "WebkitTransition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "transform": "translateY(-100%)",
            "transition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "visibility": "hidden",
          }
        }
      >
        <div
          className="bui-select-option bui-select-option-active"
          onClick={[Function]}
        >
          option 1
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 2
        </div>
        <div
          className="bui-select-option bui-select-option-disabled"
          onClick={[Function]}
        >
          option 3
        </div>
      </div>
    </div>
  </div>
  <div
    className="bui-backdrop bui-backdrop-invisible bui-select-backdrop"
    onClick={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTransition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "opacity": "0",
        "transition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "visibility": "hidden",
      }
    }
  />
</div>
`;

exports[`Select snapshot Select demo snapshot 0 6`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "stretch",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "background": "#eee",
      "padding": "50px",
    }
  }
>
  <div
    className="bui-select"
    onClick={[Function]}
  >
    <div
      className="bui-select-selector-container"
    >
      <div
        className="bui-select-selector"
      >
        自定义图标
      </div>
      <input
        className="bui-select-input"
        readOnly={true}
        value=""
      />
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"m48 55.1 21.1-22.8c1.5-1.6 4-1.7 5.7-.2 1.6 1.5 1.7 4 .2 5.7l-24 26a4.05 4.05 0 0 1-5.9 0l-24-26c-1.5-1.6-1.4-4.2.2-5.7 1.6-1.5 4.2-1.4 5.7.2l21 22.8z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-select-option-container"
      style={
        Object {
          "WebkitTransition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "opacity": "0",
          "transition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "visibility": "hidden",
        }
      }
    >
      <div
        className="bui-select-option-main"
        style={
          Object {
            "WebkitTransform": "translateY(-100%)",
            "WebkitTransition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "transform": "translateY(-100%)",
            "transition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "visibility": "hidden",
          }
        }
      >
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 1
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 2
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 3
        </div>
      </div>
    </div>
  </div>
  <div
    className="bui-backdrop bui-backdrop-invisible bui-select-backdrop"
    onClick={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTransition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "opacity": "0",
        "transition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "visibility": "hidden",
      }
    }
  />
</div>
`;

exports[`Select snapshot Select demo snapshot 0 7`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "stretch",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "background": "#eee",
      "padding": "50px",
    }
  }
>
  <div
    className="bui-select"
    onClick={[Function]}
  >
    <div
      className="bui-select-selector-container"
    >
      <div
        className="bui-select-selector"
      />
      <input
        className="bui-select-input"
        readOnly={true}
        value={2}
      />
      <svg
        className="bui-svg-icon bui-select-selector-icon"
        color="#9c9ca5"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M50 58.1c-.1.2-.3.3-.5.5-1.1.8-2.6.6-3.4-.5L32.5 40.8c-.3-.4-.5-.9-.5-1.4 0-1.3 1.1-2.4 2.5-2.4h27.1c.5 0 1.1.2 1.5.5 1.1.8 1.3 2.3.5 3.3L50 58.1z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-select-option-container"
      style={
        Object {
          "WebkitTransition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "opacity": "0",
          "transition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "visibility": "hidden",
        }
      }
    >
      <div
        className="bui-select-option-main"
        style={
          Object {
            "WebkitTransform": "translateY(-100%)",
            "WebkitTransition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "transform": "translateY(-100%)",
            "transition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "visibility": "hidden",
          }
        }
      >
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          选项 - 
          option 1
        </div>
        <div
          className="bui-select-option bui-select-option-active"
          onClick={[Function]}
        >
          选项 - 
          option 2
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          选项 - 
          option 3
        </div>
      </div>
    </div>
  </div>
  <div
    className="bui-backdrop bui-backdrop-invisible bui-select-backdrop"
    onClick={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTransition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "opacity": "0",
        "transition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "visibility": "hidden",
      }
    }
  />
</div>
`;

exports[`Select snapshot Select demo snapshot 0 8`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "stretch",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "background": "#eee",
      "padding": "50px",
    }
  }
>
  <div
    className="bui-select"
    onClick={[Function]}
  >
    <div
      className="bui-select-selector-container"
    >
      <div
        className="bui-select-selector"
      >
        <div
          className="bui-stack"
          style={
            Object {
              "--align-items": "center",
              "--flex-direction": "row",
              "--flex-wrap": "nowrap",
              "--justify-content": "center",
              "--spacing": "10px",
            }
          }
        >
          <svg
            className="bui-svg-icon"
            dangerouslySetInnerHTML={
              Object {
                "__html": "<path d=\\"m88.5 51.8-17.1 31c-1.2 2.2-3.5 3.6-6.1 3.6h-34c-2.5 0-4.8-1.4-6.1-3.6l-17.1-31c-1.2-2.1-1.2-4.7 0-6.8l17.1-31c1.2-2.2 3.5-3.6 6.1-3.6h34c2.5 0 4.8 1.4 6.1 3.6l17.1 31c1.2 2.1 1.2 4.7 0 6.8zm-5.2-2.9c.2-.3.2-.7 0-1l-17.1-31c-.2-.3-.5-.5-.9-.5H31.5c-.4 0-.7.2-.9.5l-17.1 31c-.2.3-.2.7 0 1l17.1 31c.2.3.5.5.9.5h33.9c.4 0 .7-.2.9-.5l17-31zM48.2 64c-8.8 0-16-7.2-16-16s7.2-16 16-16 16 7.2 16 16-7.2 16-16 16zm0-6c5.5 0 10-4.5 10-10s-4.5-10-10-10-10 4.5-10 10 4.4 10 10 10z\\"/>",
              }
            }
            focusable="false"
            viewBox="0 0 96 96"
          />
          option 2
        </div>
      </div>
      <input
        className="bui-select-input"
        readOnly={true}
        value={2}
      />
      <svg
        className="bui-svg-icon bui-select-selector-icon"
        color="#9c9ca5"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M50 58.1c-.1.2-.3.3-.5.5-1.1.8-2.6.6-3.4-.5L32.5 40.8c-.3-.4-.5-.9-.5-1.4 0-1.3 1.1-2.4 2.5-2.4h27.1c.5 0 1.1.2 1.5.5 1.1.8 1.3 2.3.5 3.3L50 58.1z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-select-option-container"
      style={
        Object {
          "WebkitTransition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "opacity": "0",
          "transition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "visibility": "hidden",
        }
      }
    >
      <div
        className="bui-select-option-main"
        style={
          Object {
            "WebkitTransform": "translateY(-100%)",
            "WebkitTransition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "transform": "translateY(-100%)",
            "transition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "visibility": "hidden",
          }
        }
      >
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          选项-
          option 1
        </div>
        <div
          className="bui-select-option bui-select-option-active"
          onClick={[Function]}
        >
          选项-
          option 2
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          选项-
          option 3
        </div>
      </div>
    </div>
  </div>
  <div
    className="bui-backdrop bui-backdrop-invisible bui-select-backdrop"
    onClick={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTransition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "opacity": "0",
        "transition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "visibility": "hidden",
      }
    }
  />
</div>
`;

exports[`Select snapshot Select demo snapshot 0 9`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "stretch",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "background": "#eee",
      "padding": "50px",
    }
  }
>
  <div
    className="bui-alert bui-alert-standard-info"
  >
    <div
      className="bui-alert-icon"
    >
      <svg
        className="bui-svg-icon bui-alert-default-icon"
        color="#148aff"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M48 8c22.1 0 40 17.9 40 40S70.1 88 48 88 8 70.1 8 48 25.9 8 48 8zm0 51c-2.2 0-4 1.8-4 4s1.8 4 4 4 4-1.8 4-4-1.8-4-4-4zm0-34c-2.2 0-4 1.8-4 4v22c0 2.2 1.8 4 4 4s4-1.8 4-4V29c0-2.2-1.8-4-4-4z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-alert-content"
    >
      当前状态：
      初始化
    </div>
  </div>
  <div
    className="bui-select"
    onClick={[Function]}
  >
    <div
      className="bui-select-selector-container"
    >
      <div
        className="bui-select-selector"
      >
        下拉选择
      </div>
      <input
        className="bui-select-input"
        readOnly={true}
        value=""
      />
      <svg
        className="bui-svg-icon bui-select-selector-icon"
        color="#9c9ca5"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M50 58.1c-.1.2-.3.3-.5.5-1.1.8-2.6.6-3.4-.5L32.5 40.8c-.3-.4-.5-.9-.5-1.4 0-1.3 1.1-2.4 2.5-2.4h27.1c.5 0 1.1.2 1.5.5 1.1.8 1.3 2.3.5 3.3L50 58.1z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </div>
    <div
      className="bui-select-option-container"
      style={
        Object {
          "WebkitTransition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "opacity": "0",
          "transition": "opacity 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
          "visibility": "hidden",
        }
      }
    >
      <div
        className="bui-select-option-main"
        style={
          Object {
            "WebkitTransform": "translateY(-100%)",
            "WebkitTransition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "transform": "translateY(-100%)",
            "transition": "transform 150ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
            "visibility": "hidden",
          }
        }
      >
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 1
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 2
        </div>
        <div
          className="bui-select-option"
          onClick={[Function]}
        >
          option 3
        </div>
      </div>
    </div>
  </div>
  <div
    className="bui-backdrop bui-backdrop-invisible bui-select-backdrop"
    onClick={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTransition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "opacity": "0",
        "transition": "opacity 195ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "visibility": "hidden",
      }
    }
  />
</div>
`;
